SConstruct

```diff
@@ -2519,7 +2519,8 @@ if doConfigure :
 			houdiniSources.remove( "src/IECoreHoudini/GR_Cortex.cpp" )
 			houdiniSources.remove( "src/IECoreHoudini/GR_CortexPrimitive.cpp" )
 			houdiniSources.remove( "src/IECoreHoudini/GUI_CortexPrimitiveHook.cpp" )
-			houdiniEnv.Append( CPPFLAGS = '-DIECOREHOUDINI_WITH_GL' )
+		else:
```

Comment:
You should be able to backout of the changes in cd93b92 due to 

> At Image Engine, I've disabled IECoreGL when building IECoreHoudini (see changes to config/ie/options). This is a temprorary workaround for the clash between OpenImageIO and Houdini dependencies. With GL enabled, libIECoreHoudini links to libIECoreGL links to libIECoreImage links to libOpenImageIO. At Image Engine, the only available libOpenImageIO suitabe for Cortex 10 links to Boost 1.61.0, whereas Houdini itself links to Boost 1.55.0, in blatant contradiction of VFX Platform 2017. By the time Cortex 10 is in a release state, Houdini 16.5 should be released, as well as several other OpenImageIO versions, so we should hopefully be able to re-enable it in time.

Reply:
Can we keep this for a little while, until we're certain that we don't need Houdini 16.0?